src/test/java/de/tum/in/www1/artemis/ModelingSubmissionIntegrationTest.java

```diff
@@ -589,7 +589,8 @@ void getModelSubmissionWithoutAssessment_noSubmittedSubmission_notFound() throws
         database.addModelingSubmission(classExercise, submission, "student1");
         database.updateExerciseDueDate(classExercise.getId(), ZonedDateTime.now().minusHours(1));
 
-        request.get("/api/exercises/" + classExercise.getId() + "/modeling-submission-without-assessment", HttpStatus.NOT_FOUND, ModelingSubmission.class);
+        var response = request.get("/api/exercises/" + classExercise.getId() + "/modeling-submission-without-assessment", HttpStatus.OK, ModelingSubmission.class);
```

Comment:
Also change the method name for this test (name contains "not found"), same below.

Reply:
resolved in https://github.com/ls1intum/Artemis/pull/5795/commits/2e9e90205be8c981b9391f6d7a1cda741952a8b5

Reply:
Same for `TextSubmissionIntegrationTest.java`

Reply:
done